test/e2e/operator_test.go

```diff
@@ -143,6 +144,21 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	err = ensureOperandNamespace()
+	if err != nil {
+		fmt.Printf("Failed to create external-dns namespace: %v\n", err)
+	}
```

Comment:
```suggestion
	if err = ensureOperandNamespace(); err != nil {
		fmt.Printf("Failed to create external-dns namespace: %v\n", err)
	}
```
A little more go idiomatic.

Reply:
Done. Thx